src/quo/components/share/share_qr_code/view.cljs

```diff
@@ -179,11 +204,13 @@
      - emoji:               Emoji in a string to show in the QR code.
      - on-legacy-press:     Callback for the legacy tab.
      - on-multichain-press: Callback for the multichain tab.
+     - watched-account?     Boolean
    `:wallet-multichain`
      - networks:            A vector of network names as keywords (`[:ethereum, :my-net, ...]`).
      - on-settings-press:   Callback for the settings button.
      - emoji:               Emoji in a string to show in the QR code.
      - on-legacy-press:     Callback for the legacy tab.
+     - watched-account?     Boolean
```

Comment:
if we compare with Figma this prop should be `type` with value `:watched-address` -> 
<img width="222" alt="Screenshot 2024-01-17 at 16 40 20" src="https://github.com/status-im/status-mobile/assets/22799766/2abab3e2-3711-4283-b827-102ff519ad84">

Can we align with figma instead it makes it much easier to maintain and understand what is matching to what in Figma 👍